nltk/stem/lancaster.py

```diff
@@ -304,7 +319,14 @@ def __applyRule(self, word, remove_total, append_string):
             word += append_string
         return word
 
+    def strip_prefix(self, word):
+        """Remove prefix from a word.
+        """
```

Comment:
It'll be good to acknowledge the original src from whoosh here by adding a docstring that includes the link: https://bitbucket.org/mchaput/whoosh/src/e344fb64067e45d47ec62dc65a75a50be51264a7/src/whoosh/lang/paicehusk.py#paicehusk.py-73